Dockerfile

```diff
@@ -2,8 +2,7 @@
 # https://github.com/containers/buildah/issues/1821
 FROM quay.io/buildah/stable:v1.9.0
 COPY extract.sh .
-RUN yum -y update && \
-      yum -y install wget
+RUN yum -y install wget
```

Comment:
Not sure I understand the reason for this change? We do not want to pull in other yum updates?

Reply:
just read the PR comments and noe I understand why we are skipping that.